.github/workflows/helmfile_production_plan.yaml

```diff
@@ -44,6 +44,11 @@ jobs:
           install-helm: yes
           helmfile-version: "v0.151.0"
 
+      - name: Install 1Pass CLI
+        run: |
+          curl -o 1pass.deb https://downloads.1password.com/linux/debian/amd64/stable/1password-cli-amd64-latest.deb
+          sudo dpkg -i 1pass.deb 
```

Comment:
Do we need to fetch the tfvar file from 1password? 👀

Reply:
Yup. This is required for the auto generation of vpn config.